httptransport/updateoperationhandler.go

```diff
@@ -114,7 +115,5 @@ func (h *UOHandler) Delete(w http.ResponseWriter, r *http.Request) {
 			Message: fmt.Sprintf("could not get update operations: %v", err),
 		}
 		je.Error(w, resp, http.StatusInternalServerError)
-		return
```

Comment:
is this a linter thing?

Reply:
I think so. Something flagged the useless returns for me, iirc